contents/blog/is-google-analytics-illegal-microsite.md

```diff
@@ -0,0 +1,142 @@
+---
+date: 2022-03-01
+title: "Google Analytics was deemed illegal in some EU countries. We built a microsite in 48 hours to capitalize on the news."
```

Comment:
Generally we should say 'use of' rather than GA itself in this article - especially when making claims about legality, we should take extra care to be precise in our language.